docs/how-to/type-mapping.rst

```diff
@@ -82,5 +82,30 @@ means you can treat the return value as if it were a dict - iterating over
 keys, values or items, retrieving objects by key, and so on.
 
 `NSPoint`, `NSSize`, and `NSRect`
----------------------------------
+ -------------------------------
```

Comment:
Is this valid RST? I think the underline has to be at least as long as the title, or you'll get a warning or error.